subt/docker/robotika/Dockerfile

```diff
@@ -9,11 +9,31 @@ RUN /usr/bin/python3 -m pip install --user --no-cache-dir \
   "pyzmq==19.0.0"
 
 RUN cd /home/developer/ && head -n -2 .bashrc > .bashrc.new && mv .bashrc.new .bashrc
+RUN sudo apt update
 
 COPY --chown=developer:developer . ./osgar/
 
 RUN source /opt/ros/melodic/setup.bash && ./osgar/subt/script/sync.sh
 
+#COPY subt_seed_node.cc ./src/subt_seed/src/subt_seed_node.cc
+#COPY CMakeLists.txt ./src/subt_seed/CMakeLists.txt
```

Comment:
This breaks the current solution based on x2.

Reply:
Shall I just remove both lines? They are not doing anything.

Reply:
Hmm. Yes, remove them. However I am still not sure what is actually going on. Running `sync.sh` copies `ros_proxy_node.cc` into `src` but the next lines copy something over `src` as well. Later on when the image is run locally, volume is mounted over `src` yet again. I am really lost :worried: